lib/ansible/modules/network/ios/ios_facts.py

```
@@ -227,7 +227,7 @@ def populate(self):
             self.facts['filesystems'] = self.parse_filesystems(data)
 
         data = self.responses[1]
-        if data:
+        if data and not 'Invalid input detected' in data:
```

Can warning messages be displayed in output if the command is not supported instead of silently ignoring it?